packages/components/src/components/assets/CoinLogo/CoinLogo.stories.tsx

```diff
@@ -1,13 +1,37 @@
-import { CoinLogo, variables } from '../../../index';
-import { storiesOf } from '@storybook/react';
-import { number, select } from '@storybook/addon-knobs';
+import styled from 'styled-components';
+import { CoinLogo as CoinLogoComponent, CoinLogoProps, variables } from '../../../index';
+import { StoryObj } from '@storybook/react';
 
-storiesOf('Assets/CoinLogos', module).add('CoinLogo', () => {
-    const coinsObject: any = {};
-    variables.COINS.forEach((coin: string) => {
-        coinsObject[coin] = coin;
-    });
-    const coinSelect = select('symbol', coinsObject, 'ada');
-    const size = number('size', 32);
-    return <CoinLogo size={size} symbol={coinSelect} />;
-});
+const Center = styled.div`
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    padding: 100px 0px;
+`;
+
+export default {
+    title: 'Assets/CoinLogos',
+    component: CoinLogoComponent,
+};
+
+export const CoinLogo: StoryObj<CoinLogoProps> = {
+    render: args => (
```

Comment:
nit: no need for `render:`, it will just work :)

Reply:
True, I thought that we want to have it centered so I kept the render method but even without that it looks great

Reply:
b46eecb78af7218a63cc6f2855edbf94b56b3967